index_test.go

```diff
@@ -415,8 +417,10 @@ func TestFileCase(t *testing.T) {
 func TestFileRegexpSearchBruteForce(t *testing.T) {
 	b := testIndexBuilder(t, nil,
 		Document{Name: "banzana", Content: []byte("x orange y")},
-		// ----------------------------------------0123456879
-		Document{Name: "banana", Content: []byte("x apple y")})
+		// -------------0123456
```

Comment:
nit: I think we can remove the comments in this test because we are not looking at offsets and lengths.